data/sql/updates/pending_db_world/FixQuest11879.sql

```diff
@@ -0,0 +1,58 @@
+(25802, 0, 1, 0, 38, 0, 100, 0, 0, 1, 0, 0, 0, 80, 2580200, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - On Data Set 0 1 - Run Script');
+
+-- Kaw
+UPDATE `creature_template` SET unit_flags=0, AIName='SmartAI' WHERE entry=25802;
+
+DELETE FROM `smart_scripts` WHERE entryorguid=25802 AND source_type=0;
+DELETE FROM `smart_scripts` WHERE entryorguid=25802*100 AND source_type=9;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(25802, 0, 0, 0, 6,  0, 100, 0, 0, 0, 0, 0, 0, 11, 46310, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,0, 'Kaw - On Death - Cast Drop War Halberd'),
+(25802, 0, 1, 0, 38, 0, 100, 0, 0, 1, 0, 0, 0, 80, 2580200, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - On Data Set 0 1 - Run Script'),
+(25802, 0, 2, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 41, 3000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - On Death - Force Despawn Self'),
+
+ 
+(25802*100, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,0, 'Kaw - Timed - Yell'),
+(25802*100, 9, 1, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 59, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,0,  'Kaw - Timed - Enable Running'),
+(25802*100, 9, 2, 0, 0, 0, 100, 1, 3000, 3000, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0,0, 3974.17, 5476.31, 35.602, 5.564, 'Kaw - Timed - Move to Moria'),
+(25802*100, 9, 3, 0, 0, 0, 100, 1, 2500, 2500, 0, 0, 0, 11, 46260, 2, 0, 0, 0, 0, 19, 25881, 50, 0, 0, 0, 0, 0,0,  'Kaw - Timed - Mount to Moria');
+
+DELETE FROM `creature_loot_template` WHERE `Entry`=25802 AND `Item`=35234 AND `Reference`=0 AND `GroupId`=0;
+
+-- Moria
+UPDATE `creature_template` SET unit_flags=0, AIName='SmartAI' WHERE entry=25881;
+
+DELETE FROM `smart_scripts` WHERE entryorguid=25881 AND source_type=0;
+DELETE FROM `smart_scripts` WHERE entryorguid=25881*100 AND source_type=9;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(25881, 0, 0, 0, 8, 0, 100, 1, 46315, 0, 0, 0, 0, 80, 25881*100, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,0,  'Moria - On spell hit - Start Event'),
```

Comment:
is this correct? when hit by these spells (charge and trample) we start the timed event?